### DISCUSSION TITLE
Voice Recording

### DISCUSSION THREAD

**Discussion by PanierMehl** [nextcord/nextcord#1178] [Ideas]
Support of Voice Recording Feature with WaveSink

**Comment by Skelmis** [comment]
Do you mean this? https://github.com/nextcord/nextcord/pull/1113

**Reply by PanierMehl** [reply]
Yeah, when will it be available?

**Reply by Skelmis** [reply]
It already is, you can change your installation target to be the branch.

If you meant released to pypi? We don't publish release timelines, however after using it you could plausibly provide a review on the PR to help move it along. I'd assume it may be aiming for 3.0, but due to the undocumented nature of the underlying Discord things it's pretty low priority.
